src/components/CompanyOverview.js

```diff
@@ -17,19 +116,164 @@ export default function CompanyOverview({ company }) {
   }, [company]);
 
   return (
-    <div className="overview">
-      <Typography className="overview-symbol" variant="h2" color="primary">
-        {companyData["Symbol"]}
-      </Typography>
-      <Typography className="overview-name" variant="h3">
-        {companyData["Name"]}
-      </Typography>
-      <Typography className="overview-industry" variant="body2">
-        {companyData["Industry"]}
+    <div>
+      <div className={classes.companyGeneral}>
+        <Typography className={classes.companySymbol} variant="h2">
+          {companyData["Symbol"]}
+        </Typography>
+        <Typography className={classes.companyName} variant="h3">
+          {companyData["Name"]}
+        </Typography>
+        <Typography className={classes.companyIndustry} variant="body2">
+          {companyData["Industry"]}
+        </Typography>
+      </div>
+      <div className={classes.companyStock}>
+        {/* <Typography className={classes.stockPrice} variant="h1">
+          ${stockData[0].close}
+        </Typography> */}
```

Comment:
Uncommenting these lines result in the following error message:
<code>TypeError: Cannot read property 'close' of undefined</code>
Any suggestion regarding this issue?

Reply:
I think this is because of the data has not been fetched yet to the `stockData`. That's why `stockData[0]` is undefined and unable to read the `close` property. Not sure about this. Probably we can make exception when the data is not fetched, we just show `unable to fetch data`?

Reply:
Ok, we will discuss this later.